extendr-api/src/wrapper/complexes.rs

```diff
@@ -1,5 +1,6 @@
 use super::scalar::{c64, Rcplx};
 use super::*;
+use extendr_ffi::{dataptr, R_xlen_t, Rcomplex, COMPLEX_GET_REGION, SEXPTYPE::CPLXSXP};
```

Comment:
Don't understand why `SEXPTYPE::CPLXSXP` is specifically imported.

Reply:
It appears required / expected by the `gen_vector_wrapper_impl` macro. Replacing it with `SEXPTYPE::CPLXSXP` results in compilation error. I'm going to keep it as is for now.